README.md

```diff
@@ -8,7 +8,7 @@ SystemJS
 Universal dynamic module loader - loads ES6 modules, AMD, CommonJS and global scripts in the browser and NodeJS.
 
 * [Loads any module format](docs/module-formats.md) with [exact circular reference and binding support](https://github.com/ModuleLoader/es6-module-loader/blob/v0.17.0/docs/circular-references-bindings.md).
-* Loads [ES6 modules compiled into the `System.register` bundle format for production](docs/production-workflows.md), maintaining circular references support.
+* Loads [ES6 modules compiled into the `SystemJS.register` bundle format for production](docs/production-workflows.md), maintaining circular references support.
```

Comment:
System.register is still called System.register, despite SystemJS naming. I know this is somewhat confusing.